### DISCUSSION TITLE
Behavior of ifuri mode with "unknown" CertStatus response from OCSP Responder

### DISCUSSION THREAD

**Discussion by nknkgithub** [strongswan/strongswan#2261] [Q&A]
What is the behavior when the "ifuri" revocation policy is set in swanctl.conf and "unknown" CertStatus response is sent from the OCSP responder?

**Comment by tobiasbrunner** [comment]
The OCSP response is ignored and there will be a fallback to CRLs. If none is available (or CRL validation is disabled), the status of the certificate is unknown so the validation will fail in strict mode, that is, if the policy is set to `ifuri`, it depends on whether the certificate contains an OCSP URI.

**Comment by nknkgithub** [comment]
Thanks @tobiasbrunner . Certificate contains OCSP URI and CRL validation is disabled. 

a) Will validation fail in **"ifuri"** mode if certStatus - "unknown" is returned from OCSP responder?
b) Will validation fail in **"relaxed"** mode if certStatus - "unknown" is returned from OCSP responder?
